vivisect/parsers/pe.py

```diff
@@ -286,7 +287,7 @@ def loadPeIntoWorkspace(vw, pe, filename=None):
                 vw.markDeadData(secbase, secbase+len(secbytes))
 
         except Exception, e:
-            print "Error Loading Section (%s size:%d rva:%.8x offset: %d): %s" % (secname,secfsize,secrva,secoff,e)
+            logging.getLogger("parsers.pe.loadPE").warning("Error Loading Section (%s size:%d rva:%.8x offset: %d): %s", secname, secfsize, secrva, secoff, e)
```

Comment:
I've come around to being totally onboard with tankbusta's syntax.  Lets go with each module which desires logging to have:

logger = logging.getLogger(__name__)

At the top, and subsequent calls to logger.<level>() wherever appropriate.  I'll begin this convention is future code updates as well.